core/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jboss.qa.noe</groupId>
         <artifactId>noe-core-parent</artifactId>
-        <version>0.17.8-SNAPSHOT</version>
+        <version>0.17.8-sgala-SNAPSHOT</version>
```

Comment:
@sgala this should be removed before the merge

Reply:
yes